db/src/main/java/com/readrops/db/entities/Feed.kt

```diff
@@ -51,4 +53,24 @@ data class Feed(
     @ColumnInfo(name = "open_in_ask", defaultValue = "1") var openInAsk: Boolean = true,
     @Ignore var unreadCount: Int = 0,
     @Ignore var remoteFolderId: String? = null,
-)
\ No newline at end of file
+)
+
+@DatabaseView("""
+    SELECT 
+        Feed.*,
+        CASE
+            WHEN Account.type = 'GREADER' OR Account.type = 'FRESHRSS' 
+                THEN sum(iif(not ItemState.read, 1, 0))
+            ELSE sum(iif(not Item.read, 1, 0))
+        END
+        AS unreadCount
+    FROM Account 
+    LEFT JOIN Feed ON Feed.account_id = Account.id
+    LEFT JOIN Item ON Item.feed_id = Feed.id
+    LEFT JOIN ItemState ON Item.remote_id = ItemState.remote_id
+    GROUP BY Feed.id
+""")
+data class FeedAndUnreadCount(
+    val unreadCount: Int = 0,
+    @Embedded val feed: Feed
+)
```

Comment:
So this view is basically the center point of the PR. It unifies how unread count is computed accross accounts.

*Note:* since account type configuration is not persisted in DB, the unread count logic is writtent by hand here so it will need tests to ensure completness if ever a new account config with `useSeparateState` is added in the future.